jetty-server/src/main/java/org/eclipse/jetty/server/HttpConnection.java

```diff
@@ -743,13 +746,13 @@ public Action process() throws Exception
             boolean useDirectByteBuffers = isUseOutputDirectByteBuffers();
             while (true)
             {
-                HttpGenerator.Result result = _generator.generateResponse(_info, _head, _header, _chunk, _content, _lastContent);
+                HttpGenerator.Result result = _generator.generateResponse(_info, _head, _header, _chunk, _content, getRemainingContent(), _lastContent);
                 if (LOG.isDebugEnabled())
                     LOG.debug("generate: {} for {} ({},{},{})@{}",
                         result,
                         this,
                         BufferUtil.toSummaryString(_header),
-                        BufferUtil.toSummaryString(_content),
+                        Arrays.toString(_content),
```

Comment:
probabably need a better dump than this.  Need a utility to get summary or detail strings of an array of buffers.

Reply:
Turns out we already have a `BufferUtil.toDetailString(ByteBuffer[])` method. I will use that.